apps/system/style/window.css

```diff
@@ -337,9 +337,15 @@
   opacity: 1;
 }
 
+.appWindow.in-task-manager.overlay.no-screenshot > .identification-overlay {
+  visibility: visible;
+  opacity: 1;
+}
+
+
 /* we don't want to show the overlay at all during the opening
    transition (from the homescreen and from the cardview) */
-#screen .appWindow.in-task-manager.overlay > .identification-overlay,
+#screen .appWindow.in-task-manager:not(.no-screenshot).overlay > .identification-overlay,
 .appWindow.enlarge.overlay > .identification-overlay,
 .appWindow.from-cardview.overlay > .identification-overlay {
```

Comment:
adding a `:not(.no-screenshot)` here should help for the transition back to the app